### ISSUE TITLE
Consecutive calls to adjust_learning_rate for the encoder

### ISSUE THREAD

**Issue by yanpeigong** [zhuduowang/Change3D#4]
<img width="913" height="149" alt="Image" src="https://github.com/user-attachments/assets/929f083c-fd98-47e3-bd90-d0ed00b239ee" />
Hi，I was going through the train_CC.py code and I noticed that adjust_learning_rate is called twice in a row for the encoder_optimizer within the same conditional block.
I was wondering if this is intentional or if it might be a typo（or the second call was actually intended to adjust the learning rate for the decoder_optimizer instead？）.
Could you please clarify this? Thank you for your time!

**Comment by zhuduowang** [comment]
Thanks for pointing this out!  The second call was intended for decoder_optimizer. I have fixed it. 😊
